cupy/cuda/memory.pyx

```diff
@@ -1,6 +1,7 @@
 import collections
 import ctypes
 import weakref
+import six
```

Comment:
Please add a blank line before this line.
`coolections`, `ctypes` and `weakref` is standard library.
`six` is related third party library.